Procfile.dev

```diff
@@ -0,0 +1,2 @@
+web: bin/rails server -p 4000
+js: yarn build --watch
```

Comment:
why do we need this js build? do we not use propshaft?
new rails + spree should be no build IMO